src/plugins/upload.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+const SocketIOFileUploadServer = require("socketio-file-upload/server");
+const Helper = require("../helper");
+const path = require("path");
+const fsextra = require("fs-extra");
+const fs = require("fs");
+
+function init(socket, client) {
+	const siofu = new SocketIOFileUploadServer();
+
+	siofu.uploadValidator = function(event, callback) {
+		const folder = Helper.getFileUploadPath();
+		fsextra.ensureDir(folder, (err) => {
+			if (err) {
+				callback(err);
+			} else {
+				siofu.dir = folder;
+				callback(true);
+			}
+		});
+	};
+
+	siofu.on("complete", (data) => {
```

Comment:
Handle `error` event (and possibly others? need to look at the source code) so we don't crash.

Reply:
Done